fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryDefaultOnEmptyPlan.java

```diff
@@ -93,9 +93,10 @@ public <M extends Message> RecordCursor<QueryResult> executePlan(@Nonnull final
                                                                      @Nonnull final EvaluationContext context,
                                                                      @Nullable final byte[] continuation,
                                                                      @Nonnull final ExecuteProperties executeProperties) {
-        return RecordCursor.orElse(cont -> getChild().executePlan(store, context, cont, executeProperties),
-                (executor, cont) -> RecordCursor.fromList(ImmutableList.of(QueryResult.ofComputed(onEmptyResultValue.eval(store, context)))),
-                continuation);
+        return RecordCursor.orElse(cont -> getChild().executePlan(store, context, cont, executeProperties.clearSkipAndLimit()),
+                        (executor, cont) ->
+                                RecordCursor.fromList(ImmutableList.of(QueryResult.ofComputed(onEmptyResultValue.eval(store, context))), cont), continuation)
+                .skipThenLimit(executeProperties.getSkip(), executeProperties.getReturnedRowLimit());
```

Comment:
Do we have any coverage of the skip case? In particular, we have a behavior change here for a query like:

```sql
SELECT count(*) FROM t SKIP 1
```

I think the old code would skip the underlying count (which returns 1 result), so it would hit the `else` branch and return a single result of 0, whereas now, this would apply the skip after the `else`, and return an empty result

Reply:
Sorry, this was a little unclear: I think the new behavior is better, but I was noting a lack of coverage if the behavior code change without any tests noticing.

Also, this should really be `OFFSET` instead of `SKIP`, but also, we don't currently support `OFFSET` in the SQL dialect. If it's not exposed, it shouldn't be an issue